packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -1965,11 +1965,32 @@ export class Grapher
     }
 
     @computed get showFacetControl(): boolean {
+        let {
+            hideFacetControl,
+            filledDimensions,
+            yColumnSlugs,
+            availableFacetStrategies,
+        } = this
+
+        // show facet control for some chart types by default
```

Comment:
Grasping this function is tough now, with all the (negated) logic 😬 
maybe this would be better:

```ts
if (this.hideFacetControl) return false

// heuristic: ...
if (!availableFacetStrategies.includes(FacetStrategy.none)) return true

const showFacetControlChartType = this.isStackedArea || this.isStackedBar || ...

return showFacetControlChartType && !hasProjection && yColumnSlugs.length <= 2
```

There's a slight change in logic in there (in that `hideFacetControl` always trumps everything else), but that should be fine I think